app/templating/summary_context.py

```diff
@@ -27,6 +27,6 @@ def build_summary_rendering_context(schema_json, answer_store, metadata):
                 if isinstance(value, str):
                     answers[answer_id] = escape(value)
 
-            groups.extend([Group(group, answers, path, metadata, url_for)])
+            groups.extend([Group(group, answer_store, path, metadata, url_for)])
```

Comment:
Changing this to use the answer_store rather than answers means answers aren't being escaped.